prism_benchmarks/run_prism_benchmarks.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+# This script runs the benchmarks for the Prism parser in the Sorbet type checker.
+# It assumes the presence of a few directories and tools:
+# - The yjit-bench directory, which contains the benchmarks for YJIT (https://github.com/Shopify/yjit-bench)
+# - Hyperfine, a command-line benchmarking tool (https://github.com/sharkdp/hyperfine)
+
+# Check if the yjit-bench directory exists
+if [ ! -d "../yjit-bench" ]; then
+  echo "Please clone the yjit-bench directory before running this script: https://github.com/Shopify/yjit-bench"
+  exit 1
+fi
+
+if [! command -v hyperfine >/dev/null 2>&1 ]; then
+  echo "Please install hyperfine before running this script: https://github.com/sharkdp/hyperfine"
+  exit 1
+fi
+
+# Enable globstar for recursive directory listing
+# shopt -s globstar
+
+# Read all files in the benchmarks directory to warm up the file system cache
+for i in {1..10}
+do
+  cat ../yjit-bench/benchmarks/**/*.rb > /dev/null
```

Comment:
This was Alex M's idea -- basically warm up the file system cache by reading the benchmark files a few times. I've found that it significantly improves the accuracy of the benchmarks on the first run!

Reply:
This step lowering the variance makes me think the # of times hyperfine is warming up may not be enough. It's not crucial but may be worth playing around with different warmup values.

Reply:
@KaanOzkan agreed. This is significantly faster than warmup loops, so I've done this for now, but I think it's worth reexamining the approach as we go.